env/env.cc

```diff
@@ -22,6 +23,20 @@ namespace rocksdb {
 Env::~Env() {
 }
 
+Status Env::NewLogger(const std::string& fname,
+                      std::shared_ptr<Logger>* result) {
+  EnvOptions options;
```

Comment:
By default EnvOptions.writable_file_max_buffer_size is 1MB. I'm not sure whether this is the best value here. I think at least we should explicitly set the value here, so that if one day the default is changed, the logger is not effected.

Reply:
I have explicitly set the writable_file_max_buffer_size to 1MB as of now and left a TODO to tune it.